SpringProject/src/main/java/com/gradproject2019/conferences/service/ConferenceServiceImpl.java

```diff
@@ -44,9 +45,11 @@ public ConferenceResponseDto saveConference(ConferenceRequestDto conferenceReque
     }
 
     private void checkNotInPast(Conference conference) {
```

Comment:
You should just pass the conferences dateTime here, no need to pass the whole conference